app/javascript/landing/Hero.js

```diff
@@ -23,14 +23,18 @@ const Hero = (props) => {
   return (
     <div className={classes.heroContent}>
       <Container maxWidth="sm">
+      
         <Typography
           component="h1"
           variant="h2"
           align="center"
           color="textPrimary"
           gutterBottom
         >
-          Crystal Swap
+          <span style={{position: "relative"}}>
+            Crystal Swap
+            <img width="40px" style={{position: "absolute", }} src="https://media.discordapp.net/attachments/526567429363597345/753846949735694436/beta-logo-png-8.png"/>
```

Comment:
I think the beta tag icon might look better at the bottom instead of the top.

Also we tend to use the material UI `makeStyles ` for styling things in components. This means you will give an element a class like `className={classes.betaTag}` then add the styles in the styles object here. https://github.com/gdomingu/crystal_swap/pull/36/files#diff-57b152ae006dac33b83c14b8dc42b3faL11-L19